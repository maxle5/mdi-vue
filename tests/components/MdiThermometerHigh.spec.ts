
import { mount } from "@vue/test-utils";
import { expect, test } from "vitest";
import MdiThermometerHigh from "../../src/components/MdiThermometerHigh.vue";

test("MdiThermometerHigh snapshot", () => {
  const wrapper = mount(MdiThermometerHigh, {});
  expect(wrapper.html()).toMatchSnapshot();
});
