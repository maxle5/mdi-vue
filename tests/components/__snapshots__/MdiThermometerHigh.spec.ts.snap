// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`MdiThermometerHigh snapshot 1`] = `
"<svg width="24" height="24" viewBox="0 0 24 24">
  <path d="M15 13V5A3 3 0 0 0 9 5V13A5 5 0 1 0 15 13M12 4A1 1 0 0 1 13 5H11A1 1 0 0 1 12 4Z" style="fill: currentColor;"></path>
</svg>"
`;
